### DISCUSSION TITLE
Transforming_elliptic_bitcoin_dataset

### DISCUSSION THREAD

**Discussion by SasikiranJ** [danielegrattarola/spektral#138] [Q&A]
Hi, is it possible to use this spektral for classification on elliptic bitcoin dataset?

**Comment by danielegrattarola** [comment]
Hi, 

Sure, you just need to write an appropriate function to load the data so that it has an adjacency matrix and a node feature matrix!

Cheers

**Reply by SasikiranJ** [reply]
If you dont mind would you please tell me what is best way to convert dataframe into respective matrices?

**Reply by danielegrattarola** [reply]
Sure, how is the dataframe structured? What columns do you have?

**Comment by SasikiranJ** [comment]
I have three dataframes, one for node features, second one is respective node lable , third one is edge mappings(which edge is connected to which).

**Comment by SasikiranJ** [comment]
I have created adjacency dataframe(n_nodes,n_nodes), label dataframe(n_nodes, ), node features dataframe(n_nodes, n_features)

**Comment by SasikiranJ** [comment]
I have to classify between licit, illicit . I have to train on small labelled dataset  and to predict on unknown instances.

**Comment by danielegrattarola** [comment]
@SasikiranJ if I understand correctly, then you simply need to convert the dataframes into np.arrays by calling `df.values`. Then, you can simply pass the three matrices (adjacency, node features and target) to the model. 

Have a look at the [examples](https://graphneural.network/examples/).
Cheers
